yarn.lock

```diff
@@ -823,6 +823,24 @@
     methods "^1.1.2"
     path-to-regexp "^6.1.0"
 
+"@microsoft/1ds-core-js@3.2.2", "@microsoft/1ds-core-js@^3.2.2":
+  version "3.2.2"
+  resolved "https://registry.yarnpkg.com/@microsoft/1ds-core-js/-/1ds-core-js-3.2.2.tgz#86c5afb3c6357f5d96c12086e9fe46fd2ebd82e7"
+  integrity sha512-YUsH2WAIBjM8d3weiwuvkhTZxBIlQs8ssPz5pSsVEzHXfqPU5Z66ZjWu3Wmmp4sF+8rk6oZel4EBEusf7xMTYw==
+  dependencies:
+    "@microsoft/applicationinsights-core-js" "2.8.3"
+    "@microsoft/applicationinsights-shims" "^2.0.1"
+    "@microsoft/dynamicproto-js" "^1.1.6"
+
+"@microsoft/1ds-post-js@^3.2.2":
+  version "3.2.2"
+  resolved "https://registry.yarnpkg.com/@microsoft/1ds-post-js/-/1ds-post-js-3.2.2.tgz#cbe5f8c1d1281c7177448ca3a48f16d4d4bdeed7"
+  integrity sha512-PXs+1XfA2lRNgLQq8w3x786hPLTnT6m599ROUCjgtg5wU7xOdfXFLpVw+5Okd9JD81ZhY4xXXuvVvtGu1/j1sg==
+  dependencies:
+    "@microsoft/1ds-core-js" "3.2.2"
+    "@microsoft/applicationinsights-shims" "^2.0.1"
+    "@microsoft/dynamicproto-js" "^1.1.6"
+
 "@microsoft/applicationinsights-analytics-js@2.6.4":
```

Comment:
1DS is always pinned to specific versions of AI. 1DS v3.2.2 is pinned to AI v2.8.3 and released this week 1DS 3.2.3 is pinned to AI v2.8.4 (released last week).

So by having different versions you will most likely end up with 2 "versions" of AI included in the shipped code.

Reply:
And as all of the AI modules are released (and pinned) together, you should probably update this dependency as well.
While is will (should) work as we support backward compatibility (older extensions working on newer core) we don't support forward compatibility (new extensions working on older core -- it generally does except in some cases where we ship a new core utility function that his not in the older core).

Reply:
Not sure I follow. This is just autogenerated by yarn when I added the module. Eventually we will remove app insights as a top level dependency so whatever 1DS bundles is what we will get.

Reply:
You probably have AI analytics packaged v2.6.4 listed in your package.json somewhere -- just suggesting that you update it as well to so that you only end up with 1 version of the AI code being used.

Reply:
We didn't update as we were worried it wouldn't work with CS 2.0 and our AIF key which isn't a normal AI key

Reply:
While there are a log of changes between 2.6.4 and 2.8.4, the serialized format for AI did not change, and summarizing the significant changes they are
- add "unload" support
- enable "fetch()" ajax tracking by default <- potential breaking if you didn't enable already
- perf and minification improvements
- support for fetch keepalive during unload
https://github.com/microsoft/ApplicationInsights-JS/blob/master/RELEASES.md